packages/application-config/src/process-config.ts

```diff
@@ -102,26 +113,36 @@ const processConfig = ({
     }
   }
 
+  const developmentConfig: ApplicationConfig['env']['__DEVELOPMENT__'] = isProd
+    ? undefined
+    : omitDevConfigIfEmpty({
+        oidc: isOidcForDevelopmentEnabled
+          ? omitEmpty({
+              authorizeUrl: [
+                mcApiUrl.protocol,
+                '//',
+                mcApiUrl.host.replace('mc-api', 'mc'),
+              ].join(''),
+              initialProjectKey: appConfig.env.development?.initialProjectKey,
+              teamId: appConfig.env.development?.teamId,
+              oAuthScopes: appConfig.oAuthScopes,
+            })
+          : undefined,
+        menuLinks: appConfig.menuLinks,
+        // @ts-expect-error: the `accountLinks` is not explicitly typed as it's only used by the account app.
+        accountLinks: appConfig.accountLinks,
```

Comment:
If defined, we now include the menu links into the `window.app.__DEVELOPMENT__` object.

Note that there is also the `accountLinks`. This is an undocumented/hidden field that is only required for the links of the account app.